From 8f15f95f7038a7e4c6ddf2a4c858277523583db3 Mon Sep 17 00:00:00 2001
From: Ding Yunliang <yunliang.ding@intel.com>
Date: Fri, 8 Jun 2018 14:40:28 +0800
Subject: [PATCH 084/129] media: intel-ipu4: elabrate subscribe event debug
 message

Message for Open Source:
The previous log was not complete and confusing for debug,
elabrate the message by adding more print fields.

Message for Internal:
[Issue/Feature] Same.
[Root Cause/Changes] Same.

Change-Id: Ied9fb418322b854c9016256339673db32fc57c0b
Tracked-On: #JICCCN-228
Signed-off-by: Ding Yunliang <yunliang.ding@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/pci/intel/ipu-isys-csi2.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/media/pci/intel/ipu-isys-csi2.c b/drivers/media/pci/intel/ipu-isys-csi2.c
index 9da8a51..c10ea32 100644
--- a/drivers/media/pci/intel/ipu-isys-csi2.c
+++ b/drivers/media/pci/intel/ipu-isys-csi2.c
@@ -198,7 +198,8 @@ static int subscribe_event(struct v4l2_subdev *sd, struct v4l2_fh *fh,
 {
 	struct ipu_isys_csi2 *csi2 = to_ipu_isys_csi2(sd);
 
-	dev_dbg(&csi2->isys->adev->dev, "sub->id %u\n", sub->id);
+	dev_dbg(&csi2->isys->adev->dev, "subscribe event(type %u id %u)\n",
+		sub->type, sub->id);
 
 	switch (sub->type) {
 	case V4L2_EVENT_FRAME_SYNC:
-- 
1.9.1

